.gitignore

```diff
@@ -1,5 +1,4 @@
 /target
-/Cargo.lock
```

Comment:
can we keep this for normal non-CI users?

Reply:
I guess we could, any particular reason for keeping it?